apps/web/app/(app)/environments/[environmentId]/surveys/templates/templates.ts

```diff
@@ -2553,3 +2643,26 @@ export const minimalSurvey: TSurvey = {
   resultShareKey: null,
   segment: null,
 };
+
+export const firstSurvey = {
+  ...customSurvey.preset,
+  questions: customSurvey.preset.questions.map(
+    (question) =>
+      ({
+        ...question,
+        type: TSurveyQuestionType.CTA,
+        headline: "You did it 🎉",
+        html: "You're all set up. Create your own survey to gather exactly the feedback you need :)",
+        buttonLabel: "Create survey",
+        buttonExternal: true,
+        buttonUrl: "https://app.formbricks.com",
```

Comment:
This needs to be dynamic for self hosters and should have the `WEBAPP_URL` instead of app.formbricks.com